doc/source/whatsnew/v0.23.0.txt

```diff
@@ -900,6 +900,7 @@ Offsets
 
 Numeric
 ^^^^^^^
+- Bug in ``DataFrame.rank()`` and ``Series.rank()`` when ``method='dense'`` and ``pct=True`` (:issue:`15630`)
```

Comment:
can you make these ``:func:`` refs, can you specify a little more what the bug is/was.